Source/ARTPresenceMessage.m

```diff
@@ -73,25 +70,21 @@ - (NSInteger)indexFromId {
     return index;
 }
 
-- (BOOL)isNewerThan:(ARTPresenceMessage *)latest {
-    if (latest == nil) {
-        return YES;
-    }
-
-    if ([self isSynthesized] || [latest isSynthesized]) {
-        return !self.timestamp || [latest.timestamp timeIntervalSince1970] <= [self.timestamp timeIntervalSince1970];
+- (BOOL)isNewerThan:(ARTPresenceMessage *)other {
+    if ([self isSynthesized] || [other isSynthesized]) {
+        return !self.timestamp || [other.timestamp timeIntervalSince1970] <= [self.timestamp timeIntervalSince1970];
```

Comment:
I decided to move it to presence to avoid further confusion and denoted params to `msg1` and `msg2` as it's in ably-go. https://github.com/ably/ably-cocoa/pull/1891/commits/7ae70b3a8efd93ae93643657d7de0c2abdf4a2a1